web/profiles/joinup/joinup.profile

```diff
@@ -312,6 +312,9 @@ function joinup_install_tasks_alter(&$tasks, $install_state) {
   $tasks['joinup_remove_simplenews_defaults'] = [
     'function' => [JoinupCustomInstallTasks::class, 'removeSimpleNewsDefaults'],
   ];
+  $tasks['joinup_delete_message_subscribe_defaults'] = [
```

Comment:
Shouldn't this go also in a post-update hook?

Reply:
Removed in https://github.com/ec-europa/joinup-dev/pull/977/commits/a8dc369548265226727894a9300a4c4c133978c1 due to the incompatibility of config_sync with deleted entities.